modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -78,7 +84,12 @@ def initialize(info = {})
       )
     )
 
-    register_options([ Opt::RPORT(445) ])
+    register_options(
+      [
+        Opt::RPORT(445),
+        OptBool.new('INLINE', [ true, 'Use inline technique to read protected keys from the registry remotely without saving the hives to disk', true ])
```

Comment:
I think we can add `conditions:` for this option so it's clear to the user when it's used. It's relevant for the ALL, SAM, CACHE and LSA actions but not the DOMAIN action, right?

Reply:
Absolutely, I'll add this.